inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -514,12 +514,21 @@ void MKLDNNROIPoolingNode::execute() {
                 arg.bin_area = 0;
                 arg.dst = &dst[n * dst_strides[0] + cb * dst_strides[1] + oh * dst_strides[2] + ow * dst_strides[3]];
             } else {
-                for (int c = 0; c < c_block; c++) {
-                    dst[n * dst_strides[0] + cb * dst_strides[1] + oh * dst_strides[2] + ow * dst_strides[3] + c] = 0;
+                for (int cbb_cur = 0; cbb_cur < cb_num; cbb_cur++) {
```

Comment:
Could you please add tests with cases which didn't work before?

Reply:
Backend tests with new infrastructure added